File: CHSH Game/ClassicalCHSH.java
import java.util.Random;
public class ClassicalCHSH {
	public static void main(String[] args) {
		Random r = new Random();
		int rounds = 100000, wins=0;
		for (int i=0; i<rounds; i++) {
			int Alice=r.nextInt(2), Bob=r.nextInt(2);
			if (Alice==0 && Bob==0) wins++;
			else if (Alice != Bob) wins++;			
		}
		System.out.println("Wins: "+wins+", Losses: "+(rounds-wins));
		System.out.println("Win Percentage: "+ ((double)wins/rounds)*100+"%");		
	}
}

File: CHSH Game/ClassicalCHSH.py
import random as r
rounds, wins = int(1e6), 0
for i in range(rounds):
    Alice, Bob = r.randint(0,1), r.randint(0,1)
    if Alice==Bob==0:
        wins+=1
    elif Alice != Bob:
        wins+=1
print(f"Wins: {wins}, Losses: {rounds-wins}")
print(f"Win Percentage: {wins/rounds*100:}%")
